platform/openide.util/src/org/openide/util/MapFormat.java

```diff
@@ -88,7 +88,7 @@
     * For common work use  <code>format(pattern, arguments) </code>.
     * @param arguments keys and values to use in the format
     */
-    public MapFormat(Map arguments) {
+    public MapFormat(Map<String, ? extends Object> arguments) {
```

Comment:
Just to be sure: this changes/fixates the generic bounds of an exported type - intentionally?